Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ruby:2.5.1-alpine
+MAINTAINER apply for legal aid team
+
+RUN apk --update add --virtual build-dependencies \
+                                    build-base \
+                                    libxml2-dev \
+                                    libxslt-dev \
+#                                    postgresql-dev \
+                                    sqlite-dev \
+
```

Comment:
I think we should remove this blank line. I get this warning:

```[WARNING]: Empty continuation line found in:
    RUN apk --update add --virtual build-dependencies                                     build-base                                     libxml2-dev                                     libxslt-dev                                     sqlite-dev                                     tzdata                                     && rm -rf /var/cache/apk/*
[WARNING]: Empty continuation lines will become errors in a future release.```

If we get used to seeing red text printed to the screen then we'll end up missing something important. Also, the message says it'll definitely break later.